modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js

```diff
@@ -48,53 +49,107 @@ import {paginationReducer} from './reducers/index.es';
  */
 const useFormSubmit = ({apiRef, containerRef}) => {
 	const {activePage, pages} = useFormState();
-	const {submittable} = useConfig();
+	const {submittable, validateCsrfTokenURL} = useConfig();
 
 	const handleFormSubmitted = useCallback(
 		(event) => {
 			event.preventDefault();
 
-			apiRef.current
-				.validate()
-				.then((validForm) => {
-					if (validForm) {
-						const liferayForm =
-							event.target.id &&
-							Liferay.Form.get(event.target.id);
-
-						const validLiferayForm = !Object.keys(
-							liferayForm?.formValidator?.errors ?? {}
-						).length;
-
-						if (!validLiferayForm) {
-							return;
-						}
-
-						if (submittable) {
-							Liferay.Util.submitForm(event.target);
-						}
-
-						Liferay.fire('ddmFormValid', {
-							formWrapperId: event.target.id,
-						});
-
-						Liferay.fire('ddmFormSubmit', {
-							formId: getFormId(
-								getFormNode(containerRef.current)
-							),
-						});
+			fetch(validateCsrfTokenURL, {
```

Comment:
@mlan0, can we add the logic to check the token validity into a separated method?